center_loss/data.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import print_function
+import torch.utils.data as data
+from PIL import Image
+import os
+import os.path
+import errno
+import torch
+import json
+import codecs
+import numpy as np
+import random
+
+class ImageClass():
```

Comment:
Can you make it inherit from `object` so that it's a new-style class in Python 2?